setup.py

```diff
@@ -40,6 +40,9 @@
 #   TPUVM_MODE=0
 #     whether to build for TPU
 #
+#   PJRT_ONLY=1
```

Comment:
are we going to default PJRT_ONLY=1 on head? I am afraid we will miss the XRT build failures/bugs after TF updates. I felt like setting this flag almost is equivalent to delete all XRT from our code base.

Reply:
Nope, this is a typo that I reverted by accident.